lib/cli-program.js

```diff
@@ -22,11 +22,8 @@ program
     opt.default
   ));
 
-program.parse(process.argv);
-
-// TBD non-functional error handling code
-// FUTURE TODO: handle missing argument error in the right place
-// (see issue #48)
-// if (!program.args.length) {
-//   program.help();
-// }
+const args = process.argv;
+if (args.length === 2) {
```

Comment:
or better yet:

```suggestion
const MIN_ARGS_LENGTH = 3;

if (args.length < MIN_ARGS_LENGTH) {
```

(and maybe declare `const MIN_ARGS_LENGTH` right after the imports above)